v6/v6_interface.ml

```diff
@@ -92,8 +92,6 @@ type errors =
 
 (** Pass error variant as exn for error handler *)
 exception V6_error of errors
-[@@deriving rpcty]
```

Comment:
What did this use to do before?

Reply:
nothing, ppxlib just complains more than ppx_deriving.